components/br-stream/Cargo.toml

```diff
@@ -52,5 +52,6 @@ online_config = { path = "../online_config" }
 resolved_ts = { path = "../resolved_ts"}
 
 [dev-dependencies]
+assert_matches = "1.5"
```

Comment:
@YuJuncen i mean the standard `assert_matches` 😅 

https://doc.rust-lang.org/std/assert_matches/macro.assert_matches.html